Development/ExploreASL_UpdateStructure.m

```diff
@@ -25,10 +25,10 @@ function ExploreASL_UpdateStructure(DataParPath, x)
 %% Check if we need to initialize ExploreASL
 if nargin>1 && ~isempty(x) && isfield(x,'D') && isfield(x.D,'ROOT')
     % skip initialization
-elseif ~isempty(DataParPath) && exist(DataParPath,'file')
-    x = ExploreASL_Initialize(DataParPath);
+elseif ~isempty(StudyRoot) && exist(StudyRoot,'dir')
+    x = ExploreASL_Initialize(StudyRoot);
 else
-    error(['DataParPath ' DataParPath ' is invalid/doesnt exist']);
+    error(['StudyRoot ' StudyRoot ' is invalid/doesnt exist']);
```

Comment:
Especially since it is in Development

Reply:
I mention this because the input is correctly incompletely managed, fine with either 1) handle the input arguments completely, or 2) remove 'x' as input if not used.

Reply:
It always uses the dataset root path now